tests/test_tally.py

```diff
@@ -0,0 +1,213 @@
+"""Continuous integration tests with OpenMC."""
+
+import logging
+from abc import ABCMeta, abstractmethod
+from functools import cached_property
+from pathlib import Path
+from typing import Sequence, Type
+
+import build123d as bd
+import numpy as np
+import openmc
+import openmc.stats
+import pytest
+import stellarmesh as sm
+
+RELATIVE_TOLERANCE_PERCENT = 2.5 / 100
+
+logger = logging.getLogger(__name__)
+
+
+class Model(metaclass=ABCMeta):
```

Comment:
I normally see people writing classes like this by inheriting from the `ABC` convenience class rather than using `metaclass=ABCMeta`